_tutorials/import-export-data.html

```diff
@@ -32,12 +32,15 @@ <h3 id="import-youtube">Import YouTube subscriptions</h3>
         <figure class="vertical">
             <img src="/img/screenshots/shot_subscriptions_export_light.png" class="img-responsive screenshot" />
             <figcaption>
-                <ul class="inside">
-                    <li>Go to the <a href="https://www.youtube.com/subscription_manager?action_takeout=1">YouTube subscriptions manager</a>.</li>
+                <ol>
+                    <li>Go to the <a href="https://takeout.google.com/takeout/custom/youtube">Google takeout manager</a>.</li>
                     <li>Log in if asked.</li>
-                    <li>A download will start. It contains your exported YouTube subscriptions.</li>
-                    <li>Go back to NewPipe and tap on the import file at the bottom of the screen. Now, select your downloaded YouTube export file.</li>
-                </ul>
+                    <li>Click on "All data included", then on "Deselect all", then select only "subscriptions" and click "OK".</li>
+                    <li>Click on "Next step" and then on "Create export".</li>
+                    <li>Click on the "Download" button after it appears and</li>
+                    <li>From the downloaded takeout zip extract the .json file. It is usually located under "YouTube and YouTube Music/subscriptions/subscriptions.json".</li>
```

Comment:
Would it be possible to add the <code> tag around `zip`, `.json` and `YouTube and YouTube Music/subscriptions/subscriptions.json`?

Reply:
the `<code>` tag